### DISCUSSION TITLE
能在网页右键菜单里加入个"添加到永不翻译名单"的选项吗？

### DISCUSSION THREAD

**Discussion by kulkatt** [immersive-translate/immersive-translate#832] [Ideas]
### 新功能 | New Feature

修改"永不翻译名单"是个非常常用的操作，目前使用起来过于繁琐

### 为什么您想要添加这些功能 | Why you want to add these features

1.开了自动翻译之后，仍然会有许多网站不需要自动翻译，每次都打开选项一个个添加操作实在太多了

### 提议 | Proposal

1.网页右键菜单项"把域名(xxx.com)添加到永不翻译名单"
2.如果域名已经在永不翻译名单里，网页右键菜单项变成"从永不翻译名单中移除"

**Comment by KnIfER** [comment]
试图启发维护团队： ![image](https://github.com/user-attachments/assets/c62e0a0e-f924-4c67-9532-cfae1bf1a1ec)

不这么做，添加菜单会折叠，只会更麻烦。

**Reply by theowenyoung** [reply]
@KnIfER 所以这是配置快捷键是吗？
